src/test/java/it/gov/pagopa/reward/service/synchronous/op/CreateTrxSynchronousServiceImplTest.java

```diff
@@ -377,8 +379,8 @@ void authorizeTransactionInitiativeNotInContainer() {
             Assertions.fail("Expected an Exception");
 
         } catch (Exception e){
-            Assertions.assertInstanceOf(TransactionSynchronousException.class, e);
-            SynchronousTransactionResponseDTO resultResponse = ((TransactionSynchronousException) e).getResponse();
+            Assertions.assertInstanceOf(InitiativeNotInContainerException.class, e);
```

Comment:
catch RewardCalculatorConflictException instead of Exception and then asserting the type